primitives/src/rescue/sponge.rs

```diff
@@ -0,0 +1,420 @@
+//! This file contains the APIs wrappers for ark-sponge
```

Comment:
Missing copyright information.

Reply:
we should automate this later in our precommit hook

Reply:
Done in https://github.com/EspressoSystems/jellyfish/pull/146/commits/abbdfadf85a826ce5ca406243f682ff6c4a050ea.